tariff/entsoe/static.go

```diff
@@ -0,0 +1,144 @@
+package entsoe
+
+import "encoding/xml"
+
+// This file contains static declarations of structs and constants.
+// Heavily derived from https://github.com/energy-forecast/go-entsoe (MIT license), many thanks!
+
+type ProcessType string
+
+const (
+	ProcessTypeDayAhead ProcessType = "A44"
+)
+
+type DomainType = string
+
+const (
+	DomainAL           DomainType = "10YAL-KESH-----5"
```

Comment:
Not sure this works here, but might try to use `enumer` to parse Type from "AL", "AT" etc?

Reply:
I've been kinda tempted to just ditch `DomainType` altogether considering it's unlikely for us to need to refer to a country domain in code - it's just going to be defined by the end user

**With that said**, what you're proposing would make it trivial to just have the end user pick a country code from our own list, rather than making them go off and find a domain, which is definitely more user friendly - I'll have a think about it

Reply:
I've added all areas from the Ensoe static areas list